server/gitrest/src/utils.ts

```diff
@@ -100,6 +118,7 @@ export class RepositoryManager {
         const isBare: any = 1;
         const repository = git.Repository.init(`${this.baseDir}/${repoPath}`, isBare);
         this.repositoryCache[repoPath] = repository;
+        winston.info("Created a new repo for owner " + owner + " reponame: " + name);
```

Comment:
I think it would be more readable, but I personally prefer being explicit about what the values are, to make the logging a little clearer to understand